qcfractal/interface/collections/biofragment.py

```diff
@@ -114,3 +115,4 @@ def submit_torsion_drives(self, options_set, torsions):
                 submissions.append(ret)
         return submissions
 
+collection_utils.register_collection(BioFragment)
```

Comment:
May as well add the empty line here so we dont have to push a change to portal later which gets built on the standalone

Reply:
Can you explain more here?

Reply:
Github is complaining about the missing blank line at the end of this file. I'd like to avoid the making a change later where we push to the standalone `QCPortal` repo which just updates empty lines. Its harmless, but since we are in here anyways